### PR TITLE
app-admin/salt: Bumps to fix bug#563508 and CVE-2015-6941

### PR DESCRIPTION
**Pull request by eliasp**
Gentoo-Bug: 563508
CVE: CVE-2015-6941 - salt: win_useradd module and salt-cloud display passwords in debug log

Package-Manager: portage-2.2.23

### CODE REVIEW

File: app-admin/salt/salt-2015.5.6.ebuild [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+PATCHES=(
+	"${FILESDIR}/${PN}-2014.7.1-remove-pydsl-includes-test.patch"
+	"${FILESDIR}/${PN}-2015.5.5-auth-tests.patch"
+	"${FILESDIR}/${PN}-2015.5.5-cron-tests.patch"
+	"${FILESDIR}/${PN}-2015.5.5-remove-buggy-tests.patch"
+)
+
+python_prepare() {
+	# this test fails because it trys to "pip install distribute"
+	rm tests/unit/{modules,states}/zcbuildout_test.py tests/unit/modules/{rh_ip,win_network}_test.py
```
Comment (jlec) [inline app-admin/salt/salt-2015.5.6.ebuild]: missing ```die```

File: app-admin/salt/salt-2015.8.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+	"${FILESDIR}/${PN}-2015.8.0-remove-buggy-tests.patch"
+	"${FILESDIR}/${PN}-2015.5.5-auth-tests.patch"
+	"${FILESDIR}/${PN}-2015.5.5-cron-tests.patch"
+	"${FILESDIR}/${PN}-2015.5.5-remove-buggy-tests.patch"
+)
+
+python_prepare() {
+	# this test fails because it trys to "pip install distribute"
+	rm tests/unit/{modules,states}/zcbuildout_test.py \
+		tests/unit/modules/{rh_ip,win_network,random_org}_test.py
```
Comment (jlec) [inline app-admin/salt/salt-2015.8.1.ebuild]: same

File: app-admin/salt/salt-2015.5.6.ebuild (hunk 1 above)
Comment (eliasp) [inline app-admin/salt/salt-2015.5.6.ebuild]: Fixed

File: app-admin/salt/salt-2015.8.1.ebuild (hunk 2 above)
Comment (eliasp) [inline app-admin/salt/salt-2015.8.1.ebuild]: Fixed as well and also in all other affected ebuilds.

**Comment by jlec** [comment]
@chutz Could you please proceed here?

**Comment by jlec** [comment]
merged.

**Pull request closed without merging**